asset/cpu_triggered_asset_utilization.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import argparse
+import getpass
+import sys
+import time
+import winreg
+
+LOGGED_USER_NAME = getpass.getuser()
+
+parser = argparse.ArgumentParser(description = 'CPU usage triggered asset utilization.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('filename', nargs ='+', action = 'store')
+parser.add_argument('-threshold', '-t', type = int, default = 15, help = 'The threshold percentage that triggers the utilization start.')
+parser.add_argument('-interval', '-i', type = int, default = 5, help = 'Interval in minutes used to check the CPU usage.')
+parser.add_argument('-asset_names', '-assets', '-an', default = None, help = 'The asset names that will be used in the utilization entries separated by commas.')
```

Comment:
I would just 
- keep long args and short args (no need for 3)
- Follow the double dash/single dash convention
- Use single letters for the short arg (-a, instead of -an), unless there is a conflict.